module/plugins/hooks/IRC.py

```diff
@@ -378,23 +377,119 @@ def event_c(self, args):
         task.setResult(" ".join(args[1:]))
         return ["INFO: Result %s saved." % " ".join(args[1:])]
 
+    def event_freeSpace(self, args):
+        return ["INFO: Free space is %.3f G." % (int(self.pyload.api.freeSpace())/1024/1024/1024)]
```

Comment:
Youd'd better import and use `format_size()` from [misc.py](https://github.com/pyload/pyload/blob/stable/module/plugins/internal/misc.py#L273) here.